client/src/containers/App.tsx

```diff
@@ -85,83 +85,126 @@ const WhoOwnsWhatRoutes: React.FC<{}> = () => {
-                <div className="App__warning old_safari_only">
-                  <Trans render="h3">
-                    Warning! This site doesn't fully work on older versions of Safari. Try a{" "}
-                    <a href="http://outdatedbrowser.com/en">modern browser</a>.
-                  </Trans>
-                </div>
+            <div className="App__header navbar">
+              <HomeLink />
+              {isDemoSite && (
+                <span className="label label-warning ml-2 text-uppercase">
+                  <Trans>Demo Site</Trans>
+                </span>
               )}
-              <div className="App__header">
-                <HomeLink />
-                {isDemoSite && (
-                  <span className="label label-warning ml-2 text-uppercase">
-                    <Trans>Demo Site</Trans>
-                  </span>
-                )}
-                <nav className="inline">
-                  <LocaleNavLink exact to={paths.home}>
-                    <Trans>Home</Trans>
-                  </LocaleNavLink>
-                  <LocaleNavLink to={paths.about}>
-                    <Trans>About</Trans>
-                  </LocaleNavLink>
-                  <LocaleNavLink to={paths.howToUse}>
-                    <Trans>How to use</Trans>
-                  </LocaleNavLink>
-                  <a href="https://www.justfix.nyc/donate">
-                    <Trans>Donate</Trans>
-                  </a>
+              <nav className="inline">
+                <span className="hide-lg">
+                  {getMainNavLinks()}
                   {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                  <a href="#" onClick={() => this.setState({ showEngageModal: true })}>
+                  <a href="#" onClick={() => setEngageModalVisibility(true)}>
                     <Trans>Share</Trans>
                   </a>
                   <LocaleSwitcher />
-                </nav>
-                <Modal
-                  showModal={this.state.showEngageModal}
-                  onClose={() => this.setState({ showEngageModal: false })}
-                >
-                  <h5 className="first-header">
-                    <Trans>Share this page with your neighbors</Trans>
-                  </h5>
-                  <SocialShare location="share-modal" />
-                </Modal>
-              </div>
-              <div className="App__body">
-                <WhoOwnsWhatRoutes />
-              </div>
+                </span>
+                <div className="dropdown dropdown-right show-lg">
+                  <button
+                    tabIndex={0}
+                    className={
+                      "btn btn-link dropdown-toggle m-2" + (isDropdownVisible ? " active" : "")
+                    }
+                    onClick={() => toggleDropdown()}
+                  >
+                    <i className={"icon " + (isDropdownVisible ? "icon-cross" : "icon-menu")}></i>
+                  </button>
```

Comment:
You might want to add `aria-label="menu"` and set the `aria-expanded` attribute to `"true"` or `"false"` here depending on whether the menu is collapsed or not.  This will ensure that on screen readers, focusing on the hamburger will say "menu" (instead of "button", which is currently the case on Edge + Narrator), and it will also tell the screenreader user whether the menu is currently open or closed.  See the [tenants2 code](https://github.com/JustFixNYC/tenants2/blob/66ebc930237bec3c687649864c5381cce4c269fe/frontend/lib/ui/navbar.tsx#L172-L173) for an example.